packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -419,35 +419,50 @@ export class AtomicSearchBox {
     this.ariaMessage = '';
   }
 
+  private makeSuggestionPart(isSelected: boolean, isDivider: boolean) {
+    let part = 'suggestion';
+    if (isSelected) {
+      part += ' active-suggestion';
+    }
+    if (isDivider) {
+      part += ' suggestion-divider';
```

Comment:
Need to add documentation for this one too

Reply:
true